test/e2e/bench/bench.test.ts

```
@@ -146,6 +208,8 @@ describe("E2E: ably bench publisher and subscriber", function () {
       // 3. Start Publisher
       const publisherPromise = new Promise<void>((resolvePublisher, rejectPublisher) => {
         // console.log("Spawning publisher command..."); // Debug
+        const pubEnv = { ...process.env };
+        delete pubEnv.ABLY_CLI_TEST_MODE;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)